app/assets/stylesheets/modules/error.scss

```diff
@@ -0,0 +1,84 @@
+.error {
+  color: $error-color;
+  height: 400px;
+  padding-top: 210px !important;
+  position: relative;
+  text-align: left;
+  a { color: $error-color; }
+
+  @include mq-tablet() {
+    padding-top: 155px !important;
+  }
+
+  @include mq(880px) {
+    background: none;
+    padding-top: 120px !important;
+  }
+
+  @include mq(320px) {
+    height: 550px;
+    padding-top: 16px !important;
+  }
+}
+
+.error-img {
+  position: absolute;
+
+  @include mq(880px) {
+    display: block;
+    left: 50%;
```

Comment:
here, this is something weird ... I mean, make i display block, absolute 50% from the left then move it 125px to the left??? jeje, it's kinda funny and it sounds like hackish/patch. If possible, review the markup, or classes, maybe all these "positioning" isn't necessary